great_expectations/rule_based_profiler/parameter_builder/unexpected_count_statistics_multi_batch_parameter_builder.py

```diff
@@ -264,6 +262,35 @@ def _build_parameters(  # PLR0915, PLR0915
         )
 
 
+def _standardize_mostly_for_single_batch(  # noqa: PLR0911
+    expectation_type: str, mostly: np.float64
+) -> np.float64:
+    """
+    Applies business logic to standardize "mostly" value for single-Batch case.
+    """
+    if expectation_type == "expect_column_values_to_be_null":
```

Comment:
I think this would read more easily as `if else` with a single return statement

Also, are we not rounding mostly to the nearest 2.5%?

Reply:
Yes that is correct. We only round for one case. Please see diagram attached.
<img width="569" alt="Screenshot 2023-07-12 at 9 57 38 AM" src="https://github.com/great-expectations/great_expectations/assets/2117313/b8ee42b1-7410-4f18-a1b4-b77d9b938813">